README.md

```diff
@@ -21,8 +21,17 @@ In it's simplest form:
 class Article < ActiveRecord::Base
   validates_liquid_of :content
   validates_presence_of_liquid_variable :content, variable: 'very_important_variable'
+  validates_presence_of_liquid_filter :content, filter: 'your_filter', max: "1"
```

Comment:
max should be integer. here and also other places.

Reply:
Thanks